Code/Framework/AzFramework/AzFramework/DocumentPropertyEditor/AggregateAdapter.cpp

```diff
@@ -440,53 +466,102 @@ namespace AZ::DocumentPropertyEditor
         return Dom::Value();
     }
 
+    Dom::Value RowAggregateAdapter::GetValueForNode(AggregateNode* aggregateNode)
+    {
+        return (
+            aggregateNode->m_allEntriesMatch || !m_generateDiffRows ? GenerateAggregateRow(aggregateNode)
+                                                                    : GenerateValuesDifferRow(aggregateNode));
+    }
+
+    Dom::Value RowAggregateAdapter::GetValueHierarchyForNode(AggregateNode* aggregateNode)
+    {
+        Dom::Value returnValue;
```

Comment:
Maybe worth adding null check for `aggregateNode` or maybe an assert, since that case would be a programmer error.

Reply:
done!